src/features/page_view_event/aggregate/index.js

```diff
@@ -111,6 +111,7 @@ export class Aggregate extends AggregateBase {
           activateFeatures(JSON.parse(responseText), this.agentIdentifier)
           this.drain()
         } catch (err) {
+          console.error(err)
```

Comment:
console clean up

Reply:
This was actually on purpose to aid troubleshooting why RUM fails

Reply:
If that's the case it would be better to put the `err` object as the second argument into the `warn(...)` call on 116 to keep it tidy